diff -Naur old/MANIFEST new/MANIFEST
--- old/MANIFEST	2013-07-07 07:36:58.000000000 -0500
+++ new/MANIFEST	2013-07-07 07:38:38.000000000 -0500
@@ -4,9 +4,7 @@
 MANIFEST
 README.rst
 logrep
-logrep.bat
 logrep.py
 setup.py
 wtop
-wtop.bat
 wtop.cfg
diff -Naur old/setup.py new/setup.py
--- old/setup.py	2013-07-05 09:04:00.000000000 -0500
+++ new/setup.py	2013-07-07 07:38:52.000000000 -0500
@@ -14,7 +14,7 @@
     version=logrep.VERSION,
 
     data_files=[(cfg_file_path, ["wtop.cfg"])],
-    scripts=["wtop", "logrep", "wtop.bat", "logrep.bat"],
+    scripts=["wtop", "logrep"],
     py_modules=["logrep"],
 
     url="https://github.com/ClockworkNet/wtop",
